pkg/proc/fncall.go

```diff
@@ -554,7 +560,16 @@ func funcCallCopyOneArg(scope *EvalScope, fncall *functionCallState, actualArg *
 	//TODO(aarzilli): autmoatic wrapping in interfaces for cases not handled
 	// by convertToEface.
 
-	formalArgVar := newVariable(formalArg.name, uint64(formalArg.off+int64(argFrameAddr)), formalArg.typ, scope.BinInfo, scope.Mem)
+	var formalArgVar *Variable
+	if formalArg.dwarfEntry != nil {
+		var err error
+		formalArgVar, err = extractVarInfoFromEntry(scope.BinInfo, scope.image(), scope.Regs, scope.Mem, formalArg.dwarfEntry)
```

Comment:
I think this is wrong for arguments that end up on the stack. `scope` here is the scope of the call so it doesn't have the correct value for any of the registers, and in particular SP is wrong, it should be `cfa`. I think you need to make a new scope from the thread by calling `GoroutineScope(thread)` right before the call to `funcCallEvalArgs` (disclaimer: this advice is untested and could be wrong).

Reply:
Actually it should be: before the call to `callOp` that comes right before the call to `funcCallEvalArgs`.

Reply:
Actually, I'm not sure about which one is the correct one.